mercury-operator/src/test/java/com/redhat/mercury/operator/controller/ServiceDomainControllerTest.java

```diff
@@ -61,28 +68,28 @@ public void beforeEach() throws IOException {
         }
 
         //Hack so that the cluster will already have a kafka broker
-        Kafka expectedKafka = getExpectedKafKa("", SERVICE_DOMAIN_CLUSTER_NAME, client.getNamespace());
-        Kafka fetchedKafka = client.resources(Kafka.class).inNamespace(client.getNamespace()).withName(SERVICE_DOMAIN_CLUSTER_NAME).get();
+        Kafka expectedKafka = getExpectedKafKa("", SERVICE_DOMAIN_CLUSTER_NAME, SERVICE_DOMAIN_CLUSTER_NAMESPACE);
```

Comment:
I prefer that you use the sdc namespace in these cases instead of the constant. It's fine to use a constant but here it will make it more evident why this value is set to the kafka instance

Reply:
The issue in this specific case is the SDC is created only afterwards, so I can't use its namespace here